src/Eris/Generator/FrequencyGenerator.php

```diff
@@ -16,6 +16,7 @@ function frequency(/*$frequencyAndGenerator, $frequencyAndGenerator, ...*/)
 class FrequencyGenerator implements Generator
 {
     private $generators;
+    private $frequencies;
```

Comment:
it seems there is a reference to `$this->frequencies` in this class, but no one is assigning it. The solution would be to fix that reference to the correct variable I guess